common/workunit/workunit.cpp

```diff
@@ -14137,23 +14137,31 @@ KeepK8sJobs translateKeepJobs(const char *keepJob)
     return KeepK8sJobs::none;
 }
 
+// NB: will fire an exception if command fails (returns non-zero exit code)
+static void runKubectlCommand(StringBuffer *output, const char *title, const char *cmd, const char *input, bool logOutput, bool logErrors)
```

Comment:
logErrors is always true - is it worth having?

Reply:
probably not, I'll remove